src/PIL/PSDraw.py

```diff
@@ -26,7 +26,7 @@
 class PSDraw:
     """
     Sets up printing to the given file. If **fp** is omitted,
```

Comment:
Let's also put this in code formatting?
```suggestion
    Sets up printing to the given file. If ``fp``is omitted,
```

Reply:
I'd prefer to do it in a separate PR, and keep this one only about enabling the intersphinx mapping specifically.

I imagine a grep for `\*\*[^ ]+\*\*` could find a lot of instances like this and would deserve its own PR anyway.